### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by Beanow**
Took a stab at an overall update of the readme.

Including #94 discord, #93 preview url, #75 credits for the domain.

epic badges included, because everyone knows this adds legitimacy to your project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
 
-[You might rather see the site in its final, rendered form](https://sfosc.org) - it's easier to read that way. 
+![SFOSC](static/logo.png)
+
+Welcome to the Sustainable Free and Open Source Community!
+This is the place to contribute content to the website, and collaborate
+on writing new social contracts, documenting business models, and more.
+
+The latest release is visible on https://sfosc.org - it's easier to read that way.<br/>
+The current `master` branch is visible on https://sfosc.org/preview
```
Comment (vsoch) [inline README.md]: One (unrelated) question - it looks like https://www.sfosc.org/ isn't working - who set up the DNS and can we fix this up?

**Comment by vsoch** [comment]
Links are working and good, LGTM.

File: README.md (hunk 1 above)
Comment (Beanow) [reply README.md]: Good find. Poking with dig and checking github pages docs, indeed that would be just DNS settings to fix. I'm assuming just Adam currently has access to this.

**Comment by vsoch** [comment]
Yeah! And then just make sure when you ask for (or update) certificates you register the www. too.

**Pull request merged**